lib/intercom/index.js

```diff
@@ -9,10 +9,12 @@ var defaults = require('defaults');
 var del = require('obj-case').del;
 var isEmail = require('is-email');
 var load = require('load-script');
+var hash = require('string-hash');
 var empty = require('is-empty');
 var alias = require('alias');
 var each = require('each');
 var when = require('when');
+var dot = require('obj-case');
```

Comment:
This does a bit more than act as a function version of `.`—it normalizes case, punctuation, etc. `find` is a better name